src/app/checkout/step-3/step-3.tpl.html

```diff
@@ -73,9 +73,9 @@
             <div>
               {{$ctrl.donorDetails.email}}
             </div>
-            <div class="form-group" ng-if="$ctrl.radioStationName">
```

Comment:
Commit 3d93b0b77a4aa9650a0e3c872463b370ff5e1ed3 should be split up and squashed with the commits it is fixing.